src/msg/async/AsyncConnection.cc

```diff
@@ -1321,7 +1321,8 @@ ssize_t AsyncConnection::_process_connection()
         async_msgr->ms_deliver_handle_fast_connect(this);
 
         // make sure no pending tick timer
-        center->delete_time_event(last_tick_id);
+        if (last_tick_id)
```

Comment:
imho, it is not necessary, right? as `last_tick_id` is always greater or equal to 1. if it's 0, `center->delete_time_event()` would be a no-op?